LocalPayment/src/androidTest/java/com/braintreepayments/api/localpayment/LocalPaymentClientTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.braintreepayments.api.localpayment
+
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.braintreepayments.api.core.BraintreeClient
+import com.braintreepayments.api.core.PostalAddress
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+class LocalPaymentClientTest {
+    private var countDownLatch: CountDownLatch? = null
+    private var braintreeClient: BraintreeClient? = null
```

Comment:
Optional - these `vars` could be defined as `lateinit` non null. This way we don't need to handle the null case in the tests.